source/chapters/appendix/mixxx_controls.rst

```diff
@@ -780,7 +780,9 @@ Any control listed above for :mixxx:cogroupref:`[ChannelN]` will work for a samp
                    [PreviewDeckN],bpm_tap
                    [SamplerN],bpm_tap
 
-   When tapped repeatedly, adjusts the :term:`tempo` of the channel to match the tapped :term:`BPM`.
+   When tapped repeatedly, adjusts the :term:`tempo` of the track on the deck to match the tapped :term:`BPM`.
+
+   .. note:: if you only want to change the :term:`rate` of the channel (instead of the :term:`BPM` of the file), use `script.bpm.tapButton(deck) <https://github.com/mixxxdj/mixxx/wiki/midi%20scripting#Helper-functions>`_ in your controller mapping instead.
```

Comment:
```suggestion
   .. note:: If you want to change the :term:`rate` of the channel (instead of the :term:`BPM` of the file), use `script.bpm.tapButton(deck) <https://github.com/mixxxdj/mixxx/wiki/midi%20scripting#Helper-functions>`_ in your controller mapping instead.
```